README.rst

```diff
@@ -29,6 +29,10 @@ One Time Setup
 
   # Set up a virtualenv using virtualenvwrapper with the same name as the repo and activate it
   mkvirtualenv -p python3.8 edx-arch-experiments
+  
+Local testing
+~~~~~~~~~~~~~
+In an lms or cms shell, ``pip install git+https://github.com/edx/edx-arch-experiments.git@mybranch``. Make sure to uninstall and reinstall after every change to your local branch.
```

Comment:
I think `pip install -e` is a better option here?

Reply:
I know about the `/src` directory in devstack that could be used, but not sure if we need instructions for both devstack and Tutor regarding where the code should live such that it is available to the shell?